### ISSUE TITLE
Possible license violation!

### ISSUE THREAD

**Issue by pejakm** [anthonycr/Lightning-Browser#153]
This guy totaly ripped Lightning's code and made a payed app, without providing its source code!
https://play.google.com/store/apps/details?id=com.browser.nowbasic

**Comment by anthonycr** [comment]
This browser is a complete kang with no changes other than aesthetics and string changes (I've decompiled his non-obfuscated code). All of the features in his changelog are fake as I have decompiled the app to see as I was skeptical and the developer (a teenager) will not admit to them being fake despite there being clear evidence in the source code. He's using extremely old code from Lightning Browser. The guy is blatantly ripping off customers with some of these insane claims:

* Web File compressor: is completely fake... he renamed the "block images from loading" option
* Now Browser Rendering engine: also completely fake... He renamed the "enable text reflow" option
* JavaScript Acceleration: fake... he renamed the "enable Javascript mode"

Moreover, all these options can be enabled in the free version of the app by pressing on the row that the checkbox lies in in the settings, the guy just hid the checkboxes from view.

I've tried commenting on the developer's xda-developer forum thread about how much of a scam he is but he brushes it off and most of the other users have ignored what I said. This clone is one of the few that really pisses me off because he doesn't follow the license and he is creating a bullshit product that is lying to users. This is the xda thread: http://forum.xda-developers.com/showthread.php?t=2672639

Additionally, the guy also kanged my alternate Thunder Browser and released it as a paid update called Now Browser Xtended. He also tried to avoid attributing me in this as well. I've given up fighting this person, but if anyone has any ideas how to stop them from ripping people off I'm game.

As a side note, it's sad seeing people getting banned from Google play left and right and then seeing some people that are being blatantly deceptive get away completely.
